docs: update community stappenplan (#1809)
nl-design-system/documentatie: docs/handboek/component-bijdragen/community-stappenplan/voor-organisaties.mdx

## Voeg de component toe aan de componenten bord

Doel: als organisatie wil je dit component graag beschikbaar maken voor hergebruik door andere teams. Door de status van de component en de documentatie beschikbaar te maken kunnen organisaties de component op nldesignsystem.nl vinden. Het kernteam kan de component dan de Community status geven.

> 1. Open het community bord via [Community Componenten bord op GitHub](https://github.com/orgs/nl-design-system/projects?query=is%3Aopen+Community+Componenten).

Reviewer: ```suggestion
1. Open het Communitybord via [Community Componenten bord op GitHub](https://github.com/orgs/nl-design-system/projects?query=is%3Aopen+Community+Componenten).
```

2. Klik op de + icoon links onderaan de pagina om een component toe te voegen.
3. Kies `Add item from repository`.

> 4. Selecteer `backlog` en zoek op de component naam.

Reviewer: ```suggestion
4. Selecteer `backlog` en zoek op de naam van de component.
```

5. Selecteer de component.
6. Kies rechts onderin `Add selected items`.

**🚩 Checkpoint**: Title

## Prefix van de verantwoordelijke organisatie

Doel: Om een component de status Community te geven is het belangrijk dat API's zoals design tokens de prefix van de verantwoordelijke organisatie hebben. Deze kolom is dus voor iedere component van de organisatie hetzelfde.

> Vul de prefix van de organisatie in, bijvoorbeeld `utrecht-` of `ams-`.

Reviewer: Het streepje achter de prefix hoeft hier niet bij, toch?

Author: Hmm daar heb je gelijk in.

Reviewer: ```suggestion
Vul de prefix van de organisatie in, bijvoorbeeld `utrecht` of `ams`.
```

**🚩 Checkpoint**: Organisatie prefix

## Naam van de component

Doel: Bij implementaties van de component op nldesignsystem.nl/componenten willen we de naam van de component in de component bibliotheek laten zien zodat developers en designers hem makkelijk kunnen vinden.

> Vul de naam van het css component in, bijvoorbeeld `{prefix-organisatie}-{naam-component}`.

Reviewer: ```suggestion
Vul de naam van het CSS component in, bijvoorbeeld `{prefix-organisatie}-{naam-component}`.
```

Door met [design tokens](/handboek/design-tokens/) te werken zorgen we ervoor dat meerdere organisaties de component van een eigen huisstijl kunnen voorzien. Minimaal zouden er design tokens beschikbaar moeten zijn om kleur en typografie beslissingen door te voeren.

Zorg ervoor dat de component gebruik maakt van de minimale design tokens.

> **Tip!** Om te controleren of er design tokens zijn toegepast kun je gebruik maken van de 'Inspect' functionaliteit van je browser. Wanneer je in de CSS verwijzingen ziet naar CSS variabelen zoals bijvoorbeeld `--{prefix-organisatie}-button-primary-action-background-color` of `--{prefix-organisatie}-button-font-family` kun je er vanuit gaan dat er design tokens zijn gebruikt.

Reviewer: ```suggestion
**Tip!** Om te controleren of er design tokens zijn toegepast, kun je  de 'Inspect' functie in je browser gebruiken. Wanneer je in de CSS verwijzingen ziet naar CSS-variabelen zoals bijvoorbeeld `--{prefix-organisatie}-button-primary-action-background-color` of `--{prefix-organisatie}-button-font-family` kun je ervan uitgaan dat er design tokens zijn gebruikt.
```

> **Tip!** Soms heeft een component geen specifieke design tokens nodig. Denk bijvoorbeeld aan een design tokens voor font-family bij de componenten Mark of Icon. Gebruik in dit soort gevallen je gezonde verstand.

Reviewer: ```suggestion
**Tip!** Soms heeft een component geen specifieke design tokens nodig. Denk bijvoorbeeld aan design tokens voor font-family bij componenten zoals Mark of Icon. Gebruik in dit soort gevallen je gezonde verstand.
```

Reviewer: ^ zelfde aanpassing als bij het andere stappenplan

Voor NL Design System zijn er afspraken gemaakt voor duidelijke en voorspelbare namen. De beschikbare design tokens van een component moeten voldoen aan [de naamgeving conventie van NL Design System](https://www.nldesignsystem.nl/handboek/design-tokens/#naamgeving). Zo is een component herbruikbaar voor alle verschillende thema's.

> Zorg ervoor dat de design tokens voldoen aan de naamgeving conventie.

Reviewer: ```suggestion
Zorg ervoor dat de design tokens voldoen aan de naamgevingsconventie.
```

Binnen het NL Design System werken we voor de documentatie met de [Creative Commons 0 licentie (CC0)](https://nldesignsystem.nl/open-source/cc0).

> Zorg ervoor dat de organisatie de CC0 licentie benoemt in het README.md bestand van de component in GitHub.

Reviewer: We spreken hier en hierboven van 'de organisatie'. Willen we niet uitgaan van jouw eigen organisatie, dus spreken in 'je'-vorm?

Author: Nee expres niet. Omdat deze community stappen bij een Estafettemodeldag ook door iemand van een andere organisatie kan worden gedaan.

Doel: Designers van andere organisaties kunnen de component overnemen en gaan gebruiken. De URL wordt zichtbaar op nldesignsystem.nl bij de documentatie van de component.

> Kopieer de URL van de component pagina door in Figma met rechtermuisknop op de component in de sidebar te klikken (Copy link to page).

Reviewer: ```suggestion
Kopieer de URL van de componentpagina door in Figma met de rechtermuisknop op de component in de sidebar te klikken en 'Copy link to page' te selecteren.
```

Doel: Organisaties die de component gebruiken kunnen zien of de huisstijl uit een eigen thema goed wordt toegepast. Nu, maar ook bij nieuwe versies van de component bibliotheek.

1. Clone de http://github.com/nl-design-system/themes repository.
2. Zorg dat je [pnpm](https://pnpm.io) hebt geinstalleerd.

> 3. Installeer de dependencies van de repository met `pnpm install`.

Reviewer: ```suggestion
3. Installeer de dependencies van de repository met het commando: `pnpm install`.
```

> 4. Zorg dat de design tokens beschikbaar zijn voor Storybook met `pnpm run build`.

Reviewer: ```suggestion
4. Maak de design tokens beschikbaar voor Storybook door `pnpm run build` uit te voeren.
```

> 5. Draai Storybook met `pnpm run storybook`.

Reviewer: ```suggestion
5. Draai Storybook met: `pnpm run storybook`.
```

#### Voeg organisatie toe aan de theme-toolkit

**Tip!**

> Is er al een bestand `packages/theme-toolkit/src/component-stories-{prefix-organisatie}.tsx`? Ga dan door naar de volgende sectie "Voeg een component toe".

Reviewer: ```suggestion
Is er al een bestand genaamd `packages/theme-toolkit/src/component-stories-{prefix-organisatie}.tsx` aanwezig? Ga in dat geval direct verder met de volgende sectie: "Voeg een component toe".
```

#### Voeg de component toe aan een config toe

1. Open de config.json van het thema van de organisatie.

> 2. Voeg de id van de story of stories toe aan `"stories": []`.

Reviewer: Moet id niet ID zijn?

Author: Weet ik niet. @Yolijn ?

Reviewer: ‘id’ komt op meerdere plekken voor in dit stappenplan. Controleer goed waar het voorkomt en pas het aan waar nodig 🙏

Optioneel: voeg het component ook toe aan het voorbeeld thema

1. Open de config.json van het Voorbeeld thema `packages/voorbeeld-design-tokens/src/config.json`.

> 2. Voeg de id van de story of stories toe aan `"stories": []`.

Reviewer: Moet id niet ID zijn?

Author: Weet ik niet. @Yolijn ?

Reviewer 2: Yes, is misschien wel mooier! Heb het aangepast

Doel: Developers kunnen de code zien en suggesties doen voor verbeteringen. Dit helpt het samenwerken aan toegankelijke en gebruiksvriendelijke componenten die voor iedereen bruikbaar zijn.

- Ga in GitHub naar de CSS van de component toe en kopieer de URL.

> - Vul deze URL in.

Reviewer: Moeten we deze URL 'plakken in het juiste veld'?

Author: Ja :)

Reviewer: ```suggestion
- Plak deze URL in het juiste veld.
```

**🚩 Checkpoint**: GitHub URL (CSS)

## Status bijgewerkt naar Available

Doel: alle teams kunnen nu zien dat de component beschikbaar is voor hergebruik door andere organsaties.

Zet checkpoint 'Status' op 'Available'.

> Breng de community en het kernteam op de hoogte via Slack. De component kan dan door het kernteam of op een volgende Estafettemodeldag Candidate worden gemaakt.

Reviewer: ```suggestion
Breng de community en het kernteam op de hoogte via Slack. De component kan daarna door het kernteam of tijdens een volgende Estafettemodeldag 'Candidate' worden gemaakt.
```
